en/includes/guides/authentication/oidc/private-key-jwt-client-auth.md

```diff
@@ -213,14 +227,32 @@ Be sure to replace the following values in the request:
     </tr>
 </table>
 
+{% if is_version == "7.0.0" %}
+
 ## Reuse tokens (optional)
 
 The `jti` (JWT ID) claim is a unique identifier included in the JWT token, which controls the reuse of the access token. By default, token reuse is disabled in {{ product_name }}. If you enable token reuse, the `jti` can be reused within its expiration period.
 
 To enable token reuse in {{ product_name }}.
 
-1. On the Asgardeo Console, go to **Login & Registration**.
-2. Under **Login Security**, click **Private Key JWT Client Authentication (OIDC)**.
-3. Switch on the toggle to enable token reuse.
+1. On the {{ product_name }} Console, go to **Organizational Settings** > **Account Security > Private Key JWT Client Authentication for OIDC**.
```

Comment:
These instructions are for IS 7? If so, the original ones are accurate aren't they? The updated ones are deprecated?

Reply:
Here only the Asgardeo console need to be replaced with {{ product_name }}. Will update the PR with that.

Reply:
Fixed with 125b6353d3ca3c1719293beaf109e4b71772ff62